subjects/java/checkpoints/almost-palindrom/README.md

```diff
@@ -0,0 +1,41 @@
+## AlmostPalindrome
+
+### Instructions
+
+In a file named `AlmostPalindrome.java` write a function `isAlmostPalindrome` that returns true if the String in parameter is almost palindrome.
+A word is considered "almost a palindrome" if removing one letter from the word makes it a palindrome.
+
+> 💡 The String in parametre will contains at least 3 letters.
```

Comment:
There are 2 small grammar mistakes:
- `will contains` -> should be `will contain`
- `parametre` -> should be `parameter`. An alternative could also be `The input String will contain...`

Reply:
done